LondonBritishLibrary/orient/BLorient2263.xml

```diff
@@ -0,0 +1,219 @@
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1ra" to="5vb"/>
+                            <title ref="LIT7032Trinity" type="incomplete">Fragment of an Amharic treatise on the Trinity</title>
+                            <note>The beginning is missing. An indeterminate number of leaves is lacking between <locus target="#1"/> and
+                                <locus target="#2"/>.</note>
+                            <incipit xml:lang="am">
+                                <supplied reason="undefined" resp="PRS8999Strelcyn">አ</supplied>ብ፡ እንደ፡ ልብ፡ ነው፡ ወልድ፡ እንደ፡ ቃል፡ ነው፡ መንፈስ፡ ቅዱስ፡ እንደ፡ እስትንፋስ፡ ነው። 
+                                ልብ፡ ቃል፡ ወእስትንፋስ፡ እንዳይለሐሐቁ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አይለዋወጡም። 
+                                ልብ፡ ቃል፡ ወእስትንፋስ፡ እንዳይለያዩ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አይለያዩም። 
+                                እምቅድመ፡ <pb n="1rb"/> ዓለም፡ እንዴኽ፡ ነበሩ፡ ዓለም፡ ከኀለፈም፡ እንዴኽ፡ ይኖላናሉ፡ ይኽም፡ ይኽት፡ ፀሐይ፡ ም፡ ሐንድ፡ ነው፡ በሐንደነቱ፡ ግን፡ 
+                                ሶስትንት፡ አለበት፡ አብ፡ ክበቡን፡ ይመስል፡ ወልድ፡ ብርሃኑ፡ ይመስል፡ መንፈስ፡ ቅዱስ፡ ሙቀቱ፡ ይመስል። አብነም፡ ስለመሰሉት፡ በክበቡ፡ እንደ፡ 
+                                ፀሐ<pb n="1va"/>ይ፡ ውሱን፡ ክበብ፡ አይዶለውም፡ የሰማዩም፡ ሰማይ፡ የዕመቁ፡ ም፡ ዕመቅ፡ የጽንፉ፡ ም፡ ጽንፍ፡ አይወሰነውም፡ ወልድነም፡ በብርሃኑ፡ 
+                                ስለመሰሉት፡ ውሱን፡ ብርሃን፡ አይዶለውም፡</incipit>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus target="#2r"/>
+                            <title ref="LIT3013RepCh293" type="incomplete">Hymn to the Church</title>
+                            <note>The beginning is missing. The poem is apparently inserted in the middle of <ref target="#ms_i1"/>.</note>
+                            <explicit xml:lang="gez">ለኪ፡ ዘአፈይከኡ፡ ስብሐተ።
+                                አምሳለ፡ ቍርባን፡ ተወካፊ፡ ሊተ።
+                                ወተረፈ፡ ቃልየ፡ ወትረ፡ ረስዪ፡ መሥዋዕተ።
+                                ስብሐት፡ ለኪ፡ ቤተ፡ ክርስቲያን፡ በኆልቈ፡ እስትንፋስ፡ ወነገር።
+                                ስብሐት፡ ለኪ፡ ቤተ፡ ክርስቲያን፡ በኆልቈ፡ አዕፅምት፡ ወጸጕር።
+                                ስብሐት፡ ለኪ፡ ቤተ፡ ክርስቲያን፡ ዘአብሐኪ፡ እግዚአብሔር።
+                                በላዕለ፡ ሥጋሁ፡ ቅዱስ፡ ወደሙ፡ ክቡር።</explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="6ra" to="12vb"/>
+                            <title>Turkish and Arabic vocabulary</title>
+                            <note>As a whole indicated by: <foreign xml:id="ti"><title>ነገረ፡ ትርኵ። ዓ።</title></foreign> written on the upper
+                                margin.</note>
+                        <msItem xml:id="ms_i3.1">
+                            <locus target="#6ra"/>
+                            <title ref="NAR0184ForeignWords">Tigrinya-Turkish Vocabulary</title>
+                            <note>Containing only eight words.</note>
+                            <incipit xml:lang="ti">ነዓ፡ ብሂል፡ ዢል፤
```

Comment:
It is possible to mark it up in a div at the end. I think, in this case, it will not be given in the contents but in the end of the screen as text. I do not like this option, because this text is somehow hidden at the end. However, I can make this change and give a shortened incipit in ms_item.

Reply:
Ok, I understand. Maybe it is better to have it encoded as you have done it, in that case. I don't know.